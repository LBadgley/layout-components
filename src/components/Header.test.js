import React from 'react';
import { shallow } from 'enzyme';
import Header from './HeaderComponent';

describe('header test', () => {
  it('renders the header', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
