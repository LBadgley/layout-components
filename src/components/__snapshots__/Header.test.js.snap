// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header test renders the header 1`] = `
<header
  style={
    Object {
      "display": "flex",
      "height": "2em",
      "justifyContent": "space-between",
      "padding": "1em",
    }
  }
>
  <img
    src="https://upload.wikimedia.org/wikipedia/commons/thumb/7/73/Ruby_logo.svg/1024px-Ruby_logo.svg.png"
    style={
      Object {
        "height": "100%",
      }
    }
  />
  <h1
    style={
      Object {
        "margin": 0,
      }
    }
  >
    My Dogs
  </h1>
</header>
`;
